lib/widget/place_holders.dart

```diff
@@ -0,0 +1,38 @@
+import 'package:flutter/material.dart';
+
+class CirclePlaceholder extends StatelessWidget {
+  const CirclePlaceholder({super.key});
+
+  @override
+  Widget build(BuildContext context) {
+    return Container(
+      width: 36.0,
```

Comment:
Optional: for reusable, we should put the width and height as parameters (default size is 36)